tests/fuzz/regression_test.go

```diff
@@ -35,7 +35,10 @@ var baseCases = [][]byte{
 
 // brokenCases contains test cases that are currently failing. These should only be added if the
 // failure is publicly disclosed!
-var brokenCases = map[string]string{}
+var brokenCases = map[string]string{
+	"6169070276837376": "https://github.com/go-yaml/yaml/issues/666",
+	"6087702507290624": "https://github.com/go-yaml/yaml/issues/768",
+}
```

Comment:
If you revert this change I think things will get into the expected state.

Reply:
thanks!